forms-flow-web/src/components/ServiceFlow/details/ServiceTaskDetails.js

```diff
@@ -165,30 +168,36 @@ const ServiceFlowTaskDetails = React.memo(() => {
        >
        <TaskHeader />
        <Tabs defaultActiveKey="form" id="service-task-details" mountOnEnter>
-         <Tab eventKey="form" title="Form">
-           <LoadingOverlay active={task?.assignee!==currentUser}
-                           styles={{
-                             overlay: (base) => ({
-                               ...base,
-                               background: 'rgba(0, 0, 0, 0.2)',
-                               cursor:"not-allowed !important"
-                             })
-                           }}>
-             {task?.assignee===currentUser?<FormEdit onFormSubmit={onFormSubmitCallback} onCustomEvent={onCustomEventCallBack}/>:<FormView showPrintButton={false}/>}
-           </LoadingOverlay>
-         </Tab>
-         <Tab eventKey="history" title="History">
-           <History applicationId={task?.applicationId}/>
-         </Tab>
-         <Tab eventKey="diagram" title="Diagram">
-           <div>
-             <ProcessDiagram
-               process_key={processKey}
-               processInstanceId={processInstanceId}
-               // markers={processActivityList}
-             />
-           </div>
-         </Tab>
+          <Tab eventKey="form" title="Form">
+            <LoadingOverlay active={task?.assignee!==currentUser}
+                            styles={{
+                              overlay: (base) => ({
+                                ...base,
+                                background: 'rgba(0, 0, 0, 0.2)',
+                                cursor:"not-allowed !important"
+                              })
+                            }}>
+              {task?.assignee===currentUser?<FormEdit onFormSubmit={onFormSubmitCallback} onCustomEvent={onCustomEventCallBack}/>:<FormView showPrintButton={false}/>}
+            </LoadingOverlay>
+          </Tab>
+          {
+          !getUserRolePermission(userRoles, STAFF_REVIEWER) 
+            &&
+            <>
+              <Tab eventKey="history" title="History">
+                <History applicationId={task?.applicationId}/>
+              </Tab>
+              <Tab eventKey="diagram" title="Diagram">
+                <div>
+                  <ProcessDiagram
+                    process_key={processKey}
+                    processInstanceId={processInstanceId}
+                    // markers={processActivityList}
+                  />
+                </div>
+              </Tab>
+            </>
+          }
```

Comment:
I wonder what happens to the `client`? Or we don't show the task tab to the client at all?